### PR TITLE
Added 22Delta ruleset

### PR DESCRIPTION
**Pull request by FrancisPlaza**

### CODE REVIEW

File: src/chrome/content/rules/22Delta.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<ruleset name="22Delta">
+
```
Comment (pipboy96) [inline src/chrome/content/rules/22Delta.xml]: Remove `\n` here.

File: src/chrome/content/rules/22Delta.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	<target host="22delta.com"/>
+	<target host="*.22delta.com"/>
+
+
```
Comment (pipboy96) [inline src/chrome/content/rules/22Delta.xml]: Keep only one `\n` here.

File: src/chrome/content/rules/22Delta.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+	<securecookie host=".+" name=".+"/>
+
+
```
Comment (pipboy96) [inline src/chrome/content/rules/22Delta.xml]: Keep only one `\n` here.

File: src/chrome/content/rules/22Delta.xml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+	<rule from="^http:" to="https:"/>
+
```
Comment (pipboy96) [inline src/chrome/content/rules/22Delta.xml]: Remove `\n` here.

File: src/chrome/content/rules/22Delta.xml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<ruleset name="22Delta">
+
+	<target host="22delta.com"/>
+	<target host="*.22delta.com"/>
```
Comment (pipboy96) [inline src/chrome/content/rules/22Delta.xml]:
```suggestion
	<target host="www.22delta.com" />
	<target host="api.22delta.com" />
	<target host="pass.22delta.com" />
```

File: src/chrome/content/rules/22Delta.xml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<ruleset name="22Delta">
```
Comment (pipboy96) [inline src/chrome/content/rules/22Delta.xml]:
```suggestion
<!--
	Incomplete certificate chain:
		- pass
-->
<ruleset name="22Delta">
```

File: src/chrome/content/rules/22Delta.xml [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+	<target host="22delta.com" />
+	<target host="www.22delta.com" />
+	<target host="api.22delta.com" />
+	<target host="pass.22delta.com" />
```
Comment (pipboy96) [inline src/chrome/content/rules/22Delta.xml]:
```suggestion
```

**Comment by FrancisPlaza** [comment]
@pipboy96 incomplete certificate chain for `pass.22delta.com` has been resolved in the server

**Comment by pipboy96** [comment]
@FrancisPlaza Are you the owner of `22delta.com`?

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**